### PR TITLE
feat: alerts continue and complete

### PR DESCRIPTION
**Pull request by Meng-Heng**
Hello! I tried updating this branch with no success. It was meant for one file only. Thank you!

This PR is ready for review.

### CODE REVIEW

**Comment by darcywong00** [comment]
I think on your fork we remove the [.github/workflows/add-to-keyman-project.yml](https://github.com/keymanapp/shared-sites/pull/46/files#diff-09e443edf17146b5a6372ff9611d0c3055bfcb5431604dbda288ca5643a3e874)

That's only supposed to run on the main keyman.com repo

**Comment by mcdurdin** [comment]
Let's get @Meng-Heng setup to commit to the keymanapp repo rather than the fork. @darcywong00 can you help him with that? It just simplifies a bunch of things.

File: _common/GFMAlerts.php [hunk 1]
```diff
@@ -41,32 +40,59 @@ protected function blockQuote($block)
...
             'warning' => '<svg class="octicon octicon-alert mr-2" viewBox="0 0 16 16" version="1.1" width="16" height="16" aria-hidden="true"><path fill="orange" d="M6.457 1.047c.659-1.234 2.427-1.234 3.086 0l6.082 11.378A1.75 1.75 0 0 1 14.082 15H1.918a1.75 1.75 0 0 1-1.543-2.575Zm1.763.707a.25.25 0 0 0-.44 0L1.698 13.132a.25.25 0 0 0 .22.368h12.164a.25.25 0 0 0 .22-.368Zm.53 3.996v2.5a.75.75 0 0 1-1.5 0v-2.5a.75.75 0 0 1 1.5 0ZM9 11a1 1 0 1 1-2 0 1 1 0 0 1 2 0Z"></path></svg>'
         );
-        $iconType .= $icon[$alertType] . $alertTitle; // return string
 
-        // check for matches of the $alertType in $icon. If there is no match, return plain text for the block.
         if(empty($icon[$alertType])) {
-            return parent::blockQuote($block);
+            return null;
         }
```
Comment (mcdurdin) [inline _common/GFMAlerts.php:46]: Shouldn't this be returning `parent::blockQuote($line)`?

File: _common/GFMAlerts.php [hunk 2]
```diff
@@ -41,32 +40,59 @@ protected function blockQuote($block)
...
+        if(preg_match('/^\>\s*\[\!(IMPORTANT|TIPS|NOTE|WARNING|CAUTION)\]/i', $line['text'])) {
+            return null;
+        }
+        
+        if(isset($block['interrupted'])) {
+            $block['element']['text'][] = [
+                'text' => ''
+            ];
+            unset($block['interrupted']);
+        }
```
Comment (mcdurdin) [inline _common/GFMAlerts.php:82]: I'm not sure what this is doing?

Comment (Meng-Heng) [reply _common/GFMAlerts.php:82]: I change it to this
```
if(isset($block['interrupted'])) {
            return $block;
        }
```
To handle the interruption:
```
> [!TIPS]
> [!IMPORTANT]

> Do NOT do this.
> This is the last GFMAlerts
```
Output:
image

**Review by mcdurdin — APPROVED** [review]

**Pull request merged**